repository/esb-connector-stackexchange.properties

```diff
@@ -0,0 +1,28 @@
+# =========================
+# Properties related to App
+# =========================
+#
+# Your Access Token (Mandatory)
+# accessToken=
+#
+# Your API key (Mandatory)
+# key=
+#
+# Site you prefer. e.g. stackoverflow, askubuntu.
+# site=
+#
+# ==========================================
+# Properties related to Placeholder posts
+#
+# Non of the below values are mandatory.
+# If you don't give them it skips every test
+# which is affected from them. See more
+# information in INTEGRATION_TEST.md
+# ==========================================
+# Site and id of the placeholder question.
+# placeHolderQSite=
+# placeHolderQId=
+#
+# Site and id of the placeholder answer.
+# placeHolderASite=
+# placeHolderAId=
```

Comment:
Why commented all the properties?
Add new line at the end

Reply:
Fix with 503a067